dasharo/clevo_nv41mz_v1.0.0.yaml

```diff
@@ -0,0 +1,78 @@
+client:
```

Comment:
Is this file actually needed? It may be always generated from v1.0.0 tag.

Reply:
No, accidentaly commited. Removed